readthedocs/core/views/hooks.py

```diff
@@ -136,91 +137,135 @@ def _build_url(url, projects, branches):
 
 @csrf_exempt
 def github_build(request):
-    """A post-commit hook for github."""
+    """GitHub webhook consumer
+
+    This will search for projects matching either a stripped down HTTP or SSH
+    URL. The search is error prone, use the API v2 webhook for new webhooks.
+    """
     if request.method == 'POST':
         try:
-            # GitHub RTD integration
-            obj = json.loads(request.POST['payload'])
```

Comment:
Roger, I suppose it's possible we have some old webhook configurations lingering then.